### PR TITLE
Word counter

### PR DESCRIPTION
**Pull request by Faisal5oo**

### CODE REVIEW

File: wordCounter.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//---------------Word Counter----------------
+module.exports = (str) => {
+  let characters = str.length;
+  console.log(`Number of characters in a string is ${characters}`);
```
Comment (Fer0zkhan) [inline wordCounter.js]: Take all console.logs() in the end of the file.

File: wordCounter.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  let words = str.split(" ").length;
+  console.log(` Numbers of words is ${words}`);
+  let spaceRemover = str.trimEnd();
+  console.log(`${spaceRemover}`);
+  let Splited = str.replace(/ +/g, "");
```
Comment (Fer0zkhan) [inline wordCounter.js]: variable name should have to start with small alphabets.

**Review by Fer0zkhan — CHANGES_REQUESTED** [review]

File: wordCounter.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ module.exports = (str) => {
   console.log(` Numbers of words is ${words}`);
   let spaceRemover = str.trimEnd();
   console.log(`${spaceRemover}`);
```
Comment (Fer0zkhan) [inline wordCounter.js]: Take console in the end of the file

Comment (Fer0zkhan) [inline wordCounter.js]: Take console in the end of the file

File: wordCounter.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ module.exports = (str) => {
   console.log(` Numbers of words is ${words}`);
```
Comment (Fer0zkhan) [inline wordCounter.js]: Take Console in the end of the file

**Review by Fer0zkhan — CHANGES_REQUESTED** [review]

File: wordCounter.js (hunk 4 above)
Comment (Faisal5oo) [reply wordCounter.js]: done

**Review by Fer0zkhan — APPROVED** [review]

**Pull request merged**